Makefile

```diff
@@ -5,9 +5,24 @@ watch: ## Start a file watcher to run tests on change. (requires: watchexec)
 .PHONY: all
 all: lint test build ## test -> lint -> build
 
+.PHONY: verify
+verify: ## Download & Verify dependencies
+	go mod download
```

Comment:
Download should be part of `deps`

Reply:
I think it should not as `go mod download` is for project dependencies and in `deps` we are downloading development time dependencies.
So, I think it would be better to keep it in `verify` before module's gosum check.
Just a suggestion.

Reply:
But verify should verify, not download
And deps should not verify, but download dependencies, i.e. project and tool dependencies
Single responsibility